modules/cookbooks/pages/rag.adoc

```diff
@@ -0,0 +1,174 @@
+= Retrival Augmented Generation (RAG)
+// tag::single-source[]
+:description: How to configure Redpanda Connect to create a RAG pipeline using Postgres and PGVector.
+
+This cookbook demonstrates how to create a Retrival Augmented Generation (RAG) pipeline using Postgres and https://github.com/pgvector/pgvector[PGVector]. We'll create two Redpanda Connect pipelines: one for creating embeddings with https://ollama.ai[Ollama^] and another for searching the data using https://github.com/pgvector/pgvector[PGVector]. We will run both pipelines together using xref:guides/streams_mode/about.adoc[streams mode].
+
+== Compute the embeddings
+
+We'll start out by defining the indexing pipeline that will take textual data from Redpanda, compute embeddings for it, and then write it into a Postgres table with a PGVector index on the embeddings column.
+
+Articles to be indexed are received over the topic `articles` and look like this:
```

Comment:
I'm not sure what you mean when you say `received over`.  I guess I'm not sure what the source of this json is but that maybe my lack of knowledge.